Content.Shared/CCVar/CCVars.cs

```diff
@@ -2068,5 +2068,13 @@ public sealed class CCVars : CVars
         /// </summary>
         public static readonly CVarDef<bool> DebugPow3rDisableParallel =
             CVarDef.Create("debug.pow3r_disable_parallel", true, CVar.SERVERONLY);
+
+        // AXOLOTL CVARS:
+
+        /// <summary>
+        /// Respawn time, how long the player has to wait in seconds after death.
+        /// </summary>
+        public static readonly CVarDef<float> RespawnTime =
```

Comment:
Please add a CVar that disables the ability for users to respawn from a certain pop on, as well as a CVar to enable/disable the functionality altogether.

Reply:
server specific CVars should be in their own static class

```cs
[CVarDefs]
public static class AxolotlCVars {
        /// <summary>
        /// Respawn time, how long the player has to wait in seconds after death.
        /// </summary>
        public static readonly CVarDef<float> RespawnTime = CVarDef.Create("game.respawn_time", 300.0f, CVar.SERVER | CVar.REPLICATED);
}
```

Reply:
I assumed there was some reason that Lank did that, but I guess not.

Reply:
> I assumed there was some reason that Lank did not do that, but I guess not.

afaict lank's is a replica of my code (or a replica of someone else's code who replicated my code) and I didn't because I simply didn't know it was an option back when I wrote it.

Reply:
> > I assumed there was some reason that Lank did not do that, but I guess not.
> 
> afaict lank's is a replica of my code (or a replica of someone else's code who replicated my code) and I didn't because I simply didn't know it was an option back when I wrote it.

https://github.com/Afterlight-3149/afterlight-3149/blob/2eb5dd3124446a00fc70414fff266c2a7c415459/Content.Shared/_Afterlight/AfterlightCVarsShared.cs
I located the original code and apparently it was in fact in its own CVar def. I have no idea how the copypaste train managed to lose this, then.

Reply:
Thanks for pointing this out, with all of the code improvements here I am probably just going to replace the CD version with this one.